control/tests/statesp_test.py

```diff
@@ -443,11 +442,39 @@ def empty(shape):
             m = np.matrix([])
             m.shape = shape
             return m
+
         np.testing.assert_array_equal(empty((0, 0)), g.A)
         np.testing.assert_array_equal(empty((0, D.shape[1])), g.B)
         np.testing.assert_array_equal(empty((D.shape[0], 0)), g.C)
         np.testing.assert_array_equal(D, g.D)
 
+    def test_consistent_dimensions_322(self):
+        true = consistent_dimensions(self.A322, self.B322, self.C322, self.D322)
```

Comment:
A tighter format might be `self.assertTrue(_check_statespace_dimensions(self.A322, self.B322, self.C322, self.D322))`